app/controllers/api/activity_sequence_ratings_controller.rb

```diff
@@ -0,0 +1,90 @@
+module Api
+  class ActivitySequenceRatingsController < ApiController
+    before_action :authenticate_api_user!
+    before_action :set_activity_sequence, only: %i[create update]
+    before_action :set_teacher, only: %i[create update]
+    before_action :set_activity_sequence_performed, only: %i[create update]
+    before_action :set_activity_sequence_rating, only: [:update]
+
+    def create
+      @activity_sequence_rating = ActivitySequenceRating.create_one_or_multiples(rating_params)
+      if @activity_sequence_rating.errors.empty?
```

Comment:
@hebertdougl não rola algo como?

```
if @ activity_sequence_rating  = ActivitySequenceRating.create_one_or_multiples(rating_params)
      render :show, status: :created
else
      render json: @activity_sequence_rating.errors, status: :unprocessable_entity
end
```

Reply:
Não rola porque o método `create_one_or_multiples` retorna a instância do objeto, nesse caso sempre retornaria true. Daria para fazer algo como:
```suggestion
      if (@activity_sequence_rating = ActivitySequenceRating.create_one_or_multiples(rating_params)).valid?
        render :show, status: :created
      else
        render json: @activity_sequence_rating.errors, status: :unprocessable_entity
      end
```
Oq acha?

Reply:
@hebertdougl pode ser também. mas eu tinha imaginando retornar no `create_one_or_multiples` o `valid?` ou `errors.empty?`